### ISSUE TITLE
Smaller crop size for Inception v3

### ISSUE THREAD

**Issue by MRLoghmani** [soeaver/caffe-model#24]
I am having a hard time getting some good results from Inception v3. I am trying to fine-tune the model on the (Washington) RGB-D Object Dataset, but the results are way worse than what I have for Inception v2 (-0.07 accuracy). @soeaver I was wondering if you have stored the pre-trained model you used to get the reported results for crop size = 235. Give the original resolution of the images, maybe a smaller crop is enough and can also make the training easier.

**Comment by soeaver** [comment]
Hi, I get the different crop size (235 to 459) results with the same model which using 299 crop size for training.
